Description: fix test paths
Author: Xavier Guimard <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2021-01-11

--- a/src/config/config-set.spec.ts
+++ b/src/config/config-set.spec.ts
@@ -651,7 +651,7 @@
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     const cs = createConfigSet({ jestConfig: { rootDir: '/root', cwd: '/cwd' } as any, resolve: null })
     const doResolve = (path: string) => cs.resolvePath(path, { throwIfMissing: false, nodeResolve: true })
-    expect(doResolve('json5')).toBe(resolve(__dirname, '../../node_modules/json5', require('json5/package.json').main))
+    expect(doResolve('json5')).toBe("/usr/share/nodejs/json5/lib/index.js")
     expect(doResolve('./bar.js')).toBe(resolve('/cwd/bar.js'))
     expect(doResolve('<rootDir>bar.js')).toBe(resolve('/root/bar.js'))
     expect(doResolve('<rootDir>/bar.js')).toBe(resolve('/root//bar.js'))
