Description: fix typescript
Author: Xavier Guimard <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2022-05-10

--- a/bs-logger/src/testing/target-mock.ts
+++ b/bs-logger/src/testing/target-mock.ts
@@ -15,6 +15,7 @@
   fatal: T
 }
 const extendArray = <T>(array: T[]): ExtendedArray<T> => {
+// @ts-ignore
   return Object.defineProperty(array, 'last', {
     configurable: true,
     get() {
@@ -29,6 +30,7 @@
   readonly stream: LogTarget['stream']
 
   constructor(public minLevel = -Infinity) {
+// @ts-ignore
     this.messages = Object.defineProperties(extendArray([] as LogMessage[]), {
       trace: { get: () => this.filteredMessages(LogLevels.trace) },
       debug: { get: () => this.filteredMessages(LogLevels.debug) },
@@ -37,6 +39,7 @@
       error: { get: () => this.filteredMessages(LogLevels.error) },
       fatal: { get: () => this.filteredMessages(LogLevels.fatal) },
     })
+// @ts-ignore
     this.lines = Object.defineProperties(extendArray([] as string[]), {
       trace: { get: () => this.filteredLines(LogLevels.trace) },
       debug: { get: () => this.filteredLines(LogLevels.debug) },
--- a/src/cli/index.ts
+++ b/src/cli/index.ts
@@ -50,6 +50,7 @@
 
   const cmd = isHelp && command !== 'help' ? help : run
 
+// @ts-ignore
   return cmd(parsedArgv, logger)
 }
 
--- a/src/legacy/compiler/ts-compiler.ts
+++ b/src/legacy/compiler/ts-compiler.ts
@@ -103,6 +103,7 @@
       /* istanbul ignore next */
       this._moduleResolutionHost = {
         fileExists: memoize(this._ts.sys.fileExists),
+// @ts-ignore
         readFile: this._cachedReadFile,
         directoryExists: memoize(this._ts.sys.directoryExists),
         getCurrentDirectory: () => this.configSet.cwd,
