Description: drop network test
Author: Yadd <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2022-09-29

--- a/scripts/post-build.js
+++ b/scripts/post-build.js
@@ -2,6 +2,7 @@
 const execa = require('execa')
 const { generatedPath, rawCompilerOptionsFileName } = require('./lib/paths')
 
+/*
 try {
   execa.sync('git', ['--version'])
   if (execa.sync('git', ['diff-index', '--name-only', 'HEAD']).stdout.includes(rawCompilerOptionsFileName)) {
@@ -12,4 +13,5 @@
 } catch (e) {
   console.log('git command is not available. Skip checking generated types')
 }
+*/
 computePackageDigest()
